plugins/aea-test-autonomy/tests/test_base_test_classes/test_agents.py

```diff
@@ -84,11 +81,6 @@ def get_package_name(cls: Type) -> str:
         assert self.test_cls.happy_path == ()
         assert self.test_cls.strict_check_strings == ()
 
-        # no overwrite of parent class setup_class -> no tests needed for setup_class
-        child, parent = self.test_cls, self.test_cls.__mro__[1]
-        assert not get_package_name(child) == get_package_name(parent)
-        assert child.setup_class.__func__ == parent.setup_class.__func__
```

Comment:
Why removed?

Reply:
since this was introduced 
https://github.com/valory-xyz/open-autonomy/blob/d3f62438ab2a0f4ade24b8bed461176477287f17/plugins/aea-test-autonomy/aea_test_autonomy/base_test_classes/agents.py#L108-L113